packages/eslint-config-airbnb/rules/style.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     'brace-style': [2, '1tbs', { 'allowSingleLine': true }],
     // require camel case names
     'camelcase': [2, { 'properties': 'never' }],
-    // enforce spacing before and after comma
```

Comment:
ah - i misunderstood your question in gitter. in fact, this is correct as-is, because "false" enforces that there is *no* space before the comma. it's "enforce spacing", not "enforce spaces".